lib/Doctrine/ORM/QueryBuilder.php

```diff
@@ -183,7 +183,7 @@ public function expr()
      *
      * @param boolean $cacheable
      *
-     * @return \Doctrine\ORM\AbstractQuery This query instance.
+     * @return QueryBuilder
```

Comment:
It is better use `@return $this` here. An explanation https://github.com/php-fig/fig-standards/pull/169/files#diff-9a10a11696dc38209c125ea9c57a565fR2195